src/ui/details/list_item.rs

```diff
@@ -26,19 +30,30 @@ use crate::{
 				EditButtonSlot, HistoryButtonSlot, ViewButtonSlot,
 			},
 			detail_view::{
-				save_edit, SaveEdit, INPUT_LINE_WIDTH, LINE_WIDTH, SECRET_PLACEHOLDER,
+				save_edit, SaveEdit, INPUT_LINE_WIDTH, LINE_WIDTH, MULTILINE_HEIGHT,
+				SECRET_MULTILINE_PLACEHOLDER, SECRET_PLACEHOLDER,
 			},
 			dyn_field_title_form::{dyn_field_title_form, DynFieldTitleForm},
 		},
 		primitives::{
 			button::{icon_button, IconButton},
 			input_button_field::{input_button_field, InputButtonField},
 			input_field::input_field,
+			multiline_input_field::multiline_input_field,
+			styles,
 			tooltip::TooltipSignals,
 		},
 	},
 };
 
+fn replace_consecutive_newlines(input: String) -> String {
+	let mut output = input.clone();
+	if input.contains("\n\n") {
+		output = replace_consecutive_newlines(input.replace("\n\n", "\n \n"));
+	}
+	output
+}
```

Comment:
Now this one was needed because floem removes all new lines from rendering and if you have a note that has a bunch of empty lines they just won't show which feels weird. Should be pretty save of a recursion though